src/lib.rs

```diff
@@ -34,6 +34,8 @@
 #![cfg_attr(all(feature = "nightly", test), feature(test))]
 #![cfg_attr(feature = "no_std", no_std)]
 
+extern crate alloc;
+
```

Comment:
We should use alloc only for ansi, and the rest should be able to be used without alloc, so alloc should be a feature.

Reply:
Fixed in https://github.com/alacritty/vte/pull/85/commits/9f3c6850adc997df324fe79b76ac624e6c6b6e29.